build_tools/cmake/test_riscv64.sh

```diff
@@ -80,3 +80,7 @@ ctest --test-dir ${BUILD_RISCV_DIR}/tests/e2e/models -R llvm-cpu_local-task_mobi
 ctest --test-dir ${BUILD_RISCV_DIR}/tests/e2e/tosa_ops -R check_llvm-cpu_local-task
 # Test all xla ops except fp16, which is not supported properly
 ctest --test-dir ${BUILD_RISCV_DIR}/tests/e2e/xla_ops -R check_llvm-cpu_local-task -E fp16
+# Test runtime unit tests
+ctest --test-dir ${BUILD_RISCV_DIR}/runtime/ --timeout 900 --output-on-failure \
```

Comment:
Can we move this all into one ctest command? I'd like to see what happens if we just run ctest on the entire test directory. The compiler tests shouldn't have been built at all and we have canonical exclusion tags for various other weird things (like fp16) We might use arrays to build up the command line, as we do in other testing scripts.

If that turns out to be not possible, I notice that the other test commands here don't have the timeout set. We should also run the unit tests *before* the e2e tests if they can't be at the same time